projects/odf_transform/test/test_headers/CTD_HUD2018030_003_01_DN.ODF.json.nc.txt

```diff
@@ -0,0 +1,1231 @@
+netcdf CTD_HUD2018030_003_01_DN.ODF.json {
+dimensions:
+	z = 62 ;
+variables:
+	string filename ;
+		string filename:_FillValue = "nan" ;
```

Comment:
This field should include the ODF file name? CTD_HUD2018030_003_01_DN.ODF

Reply:
The DN in the file name indicates that this is a downward profile as oppose to an upward profile.  This should be indicated somewhere in the header.  The upward CTD profile has UP in the file name.  Often people are more interested in the Downward profile data since the upward profile is where the bottles are fired and the CTD periodically stops at several depths.